gemrb/plugins/SDLAudio/SDLAudio.cpp

```diff
@@ -322,22 +325,35 @@ Holder<SoundHandle> SDLAudio::Play(StringView ResRef, unsigned int channel,
 		*length = time_length;
 	}
 
-	Mix_VolumeChunk(chunk, MIX_MAX_VOLUME * (GetVolume(channel) * volume / 10000.0f));
+	Mix_VolumeChunk(chunk, MIX_MAX_VOLUME * GetVolume(channel) / 100);
```

Comment:
This looks like a mistake, now it's only taking the preset relative volume into account, not the user settings.

Reply:
It's taking preset relative volume into account while mixing the audio chunk.
User setting is taken into account on the line 335 for the channel itself
`Mix_Volume(chan, MIX_MAX_VOLUME * volume / 100);`